src/styles/product-card.scss

```diff
@@ -0,0 +1,111 @@
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+:root {
+  --color-for-main-text: #060B35;
+  --color-for-secondary-text: #616070;
+  --color-for-button: #00ACDC;
+  --color-for-background: #fff;
+  --color-for-button-text: #fff;
```

Comment:
Use sass vars instead and then move them to separate file and also why do you need clone 2 last vars twice, delete one